.circleci/config.yml

```diff
@@ -12,11 +12,23 @@ commands:
   prepare_docker:
     description: "Docker login"
     steps:
+      - run:
+          name: Enable experimental Docker features
+          command: export DOCKER_CLI_EXPERIMENTAL=enabled
       - run:
           name: Docker login
           command: |
+            echo "Login in $DOCKER_REGISTRY with user $DOCKER_USER"
             echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin $DOCKER_REGISTRY
-            echo $DOCKER_PASS_2 | docker login -u $DOCKER_USER_2 --password-stdin $DOCKER_REGISTRY_2
+            echo "Login in $DOCKER_REGISTRY_2 with user $DOCKER_USER_RW"
+            echo $DOCKER_PASSWORD_RW | docker login -u $DOCKER_USER_RW --password-stdin $DOCKER_REGISTRY_2
+      - run:
+          name: DCT signing setup
+          command: |
+            mkdir -p $HOME/.docker/trust/private
+            echo $DCT_KEY | base64 --decode > $HOME/.docker/trust/private/$DCT_HASH.key
```

Comment:
Why is the key named DCT_HASH once decoded ?

Reply:
Those values are being defined in the `dockerhub-opsquorum-dct` CircleCI context by Joshua, I guess, for the opsquorum team. So it is nothing I can do here to rename it